### DISCUSSION TITLE
Need some help with puzzling new Mame launch slowdown behaviour.

### DISCUSSION THREAD

**Discussion by john-iv** [mamedev/discussions#29] [Q&A]
I had to rebuild my main system last week (Win11 x64 w/ 64GB of 5200 DDR5 RAM) and have noticed that the nightly mame.exe's I produce w/ bleeding edge compile tools (pacman -Syu on mame dist package) now take about 5 times longer to launch than before.

make -Werror STRIP_SYMBOLS=1 REGENIE=1 -j21

Previously for example I could launch 'mame dkong -v' from command prompt and immediately see text in console as the parsing started.
Now when I do that I get no console output for about 5-8 seconds until emulation starts.  It's almost like the .exe doesn't get into the Windows memory cache system for subsequent 'quicker' starts.

Interestingly, mamedev official release .251 does not have this problem and starts immediately as before.

I believe my system settings are the same as before the rebuild, including Defender exclusions for the mame.exe and its process just in case.  I have also tried building w/ the original 1/22 mamedev.org compile package just to take that out of the equation.

I've never seen this behaviour before in the last 25 years.  Any ideas?

Additionally, is there a place in GITHUB to download a nightly/daily binary for tip of master?  Thanks -

**Comment by startaq** [comment]
> Additionally, is there a place in GITHUB to download a nightly/daily binary for tip of master?

https://github.com/mamedev/mame/actions

Left side select CI matching your OS, then select the build you want and download artifact.

**Reply by john-iv** [reply]
Thanks Dirk, unfortunately the last artifact build .exe exhibits the same problem on my system. Hmm.

**Reply by jflatt** [reply]
I"ve seen similar slowdown in the past when my mame.ini file path settings were incorrect

**Comment by john-iv** [comment]
Thanks @jflatt I should follow this up; somewhere along the line this corrected itself.  When it was an issue I created a RAM drive on my 64GB system and threw MAME up into it and that alleviated the problem at the time.  Will close this discussion, thanks.
